apps/desktop/electron/database/database.services.ts

```diff
@@ -117,13 +118,22 @@ export function connect() {
 
 /* ============================ Handlers ============================ */
 async function connectWrapper<T>(
-    func: (args: any) => DatabaseResponse<T | undefined>,
+    func: (args: any) => DatabaseResponse<T | undefined> | T,
     args: any = {},
 ): Promise<DatabaseResponse<T | undefined>> {
     const db = connect();
+    const orm = getOrm(db);
     let result: Promise<DatabaseResponse<T | undefined>>;
     try {
-        result = Promise.resolve(func({ ...args, db }));
+        let fnResult = func({ ...args, db, orm });
+        if (
```

Comment:
The DatabaseResponse wrapper wasn't doing anything for me, since it was always 
```
{ success: true, data: T }
```
so I made it so I could just return T.